functions.php

```diff
@@ -27,5 +27,8 @@
   require_once($filepath);
 }
 
+// Custom image size for header logo
+add_image_size( 'logo', 194, 88 );
```

Comment:
If we're gonna add this in now, we'll want to add this to `lib/init.php` and have it execute as part of the `after_setup_theme` hook. With the setup we have here, `functions.php` is just a loader. 

I'll add some documentation to make this setup a bit more clear